src/player.h

```diff
@@ -920,6 +920,7 @@ class player : public Character, public JsonSerializer, public JsonDeserializer
         int power_level, max_power_level;
         int hunger, thirst, fatigue;
         int stomach_food, stomach_water;
+		int tank_plut, reactor_plut, slow_rad;
```

Comment:
eek, tabs.  Check your text editor: most can set the tab key to insert spaces instead, so indents are uniform between editors.